project/HaHaFi/proposal_1/README.md

```diff
@@ -0,0 +1,67 @@
+#  LSD (Line Segment Detector)
+## Introdution
+In computer vision region, detection of the line on the images in real time is an useful function, because it has a lot of downstream task, such as vanish point method base camera calibration and road marking segmentation.
+
+![](https://i.imgur.com/WD8w34b.png)
+
+reference from [2](https://www.ipol.im/pub/art/2012/gjmr-lsd/article.pdf)
+## Problem to solve
+1. The time of line detection should be as fast as possible.
+2. The detection result should be visualized.
+
+
+## Prospective Users
+The people who want to get line information on the image and need a fast algorithm. 
+
+
+## System Architecture
+OS imformation: ubuntu 22.04
```

Comment:
It's good to specify the OS to run on.  When talking about performance, hardware platform matters too.

Reply:
Sure, I will add architectures information.